TestAssets/DesktopTestProjects/BindingRedirectSample/AppWithRedirectsAndConfig/AppWithRedirectsAndConfig.csproj

```diff
@@ -0,0 +1,57 @@
+﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
@krwq may have used an older build's migration tool. It would be good to re-migrate so we have the cleanest available csproj...